index.html

```diff
@@ -297,8 +297,8 @@ <h4 class="strong-title wow fadeInDown" data-wow-delay=".5s" lang="en">We've cre
     </div>
     <div id="contribute-details"></div>
   </section>
-  <section id="opensuse-conferences">
-    <div class="section-background"></div>
+  <section id="opensuse-conferences hidden">
+    <div class="section-background hidden-content"></div>
```

Comment:
This change will hide the "meet the openSUSE community at conferences" section - I slightly ;-) doubt that you want that.

BTW: if you are unsure if/how your changes work, you can easily test them - press F12 in Firefox, and you can edit the page on the fly (and do some other helpful things).